node/lib/util/status_util.js

```diff
@@ -613,3 +613,36 @@ Before proceeding, you must complete the ${command} in progress (by running
 'git meta ${command} --abort').`);
     }
 };
+
+/**
+ * Return an error message if the specified `status` of a repository isn't in a
+ * normal, ready state, that is, it does not have any conflicts or in-progress
+ * operations from the sequencer.  Adjust output paths to be relative to the
```

Comment:
I have no idea what the "adjust output paths" sentence means but it doesn't seem to relate to anything in this function?

Reply:
me neither, think this patch will go away after rebase